OJP/OJP_Fare.xsd

```diff
@@ -190,6 +222,29 @@
 			<xs:element name="Extension" type="xs:anyType" minOccurs="0"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="PlaceFareResultStructure">
+		<xs:annotation>
+			<xs:documentation>PLACE-related Fare information.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Place" type="PlaceStructure">
+				<xs:annotation>
+					<xs:documentation>The involved PLACE. Usually a StopPoint, StopPlace or PointOfInterest.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="FareProduct" type="FareProductStructure" minOccurs="0" maxOccurs="unbounded">
```

Comment:
Would it make sense to have cardinality 1, such that each PlaceFareResult is always for exactly one FareProduct?

Reply:
i think it should stay that way.  A Place can have multiple FareProducts e.g. Parking 1h, Parking 24hh, Parking_for_residents, Parking_with-subscription.  I think they can be in one PlaceFareResult. We do this this way also in TripFareResult (e.g. 1. and 2. class)